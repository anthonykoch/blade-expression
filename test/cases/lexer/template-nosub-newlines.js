`hello\n\n`
123
