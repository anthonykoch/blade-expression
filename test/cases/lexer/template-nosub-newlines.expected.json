---
name: Template - no substitution
description: nosub template with newlines
---

[
  { "type": 10, "value": "`hello\\n\\n`", "line": 1, "column": 0, "start": 0, "end": 11  },
  { "type": 0,  "value": "123",           "line": 2, "column": 0, "start": 12, "end": 15 }
]
